src/mixin/image.js

```diff
@@ -12,6 +12,10 @@ export default {
     blur: {
       type: Number
     },
+    aspectRatio: {
+      type: Number,
+      default: 0
```

Comment:
we don't need to add a default value for this prop.
without a value we can just check for an defined value instead of checking if the value is major than 0